### PR TITLE
Update example to enable logging

### PR DESCRIPTION
**Pull request by blublinsky**
/## What changes were proposed in this pull request?
With the latest version of cloudflow (2.0.9), the log stopped working.
This seems to be related to Akka upgrade. The logger used here is from Akka logging
https://doc.akka.io/docs/akka/current/logging.html
The documentation https://doc.akka.io/docs/akka/current/logging.html#loggers suggest using Akka configuration to enable 
SLF, but it did not work for me, hence this change.
This is just a quick fix to make it work, but further investigation is required.

### Why are the changes needed?
To enable working when running on the cluster. Local run still works

### Does this PR introduce any user-facing change?
no

### How was this patch tested?
Running both on cluster and locally to ensure that it works

### CODE REVIEW

**Comment by blublinsky** [comment]
One question. Do you have application.conf somewhere, which I could not overwrite?

File: examples/snippets/modules/ROOT/examples/sensor-data-scala/build.sbt [hunk 1]
```diff
@@ -14,7 +14,7 @@ lazy val sensorData =  (project in file("."))
       libraryDependencies ++= Seq(
         "com.lightbend.akka"     %% "akka-stream-alpakka-file"  % "1.1.2",
         "com.typesafe.akka"      %% "akka-http-spray-json"      % "10.1.12",
-        "ch.qos.logback"         %  "logback-classic"           % "1.2.3",
+//        "ch.qos.logback"         %  "logback-classic"           % "1.2.3",
```
Comment (RayRoestenburg) [inline examples/snippets/modules/ROOT/examples/sensor-data-scala/build.sbt:17]: Always remove commented lines please.

File: examples/snippets/modules/ROOT/examples/sensor-data-scala/src/main/resources/log4j.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+                   value="%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %m%n" />
+        </layout>
+    </appender>
+
+    <root>
+        <level value="DEBUG" />
+        <appender-ref ref="console" />
+    </root>
+
+</log4j:configuration>
```
Comment (RayRoestenburg) [inline examples/snippets/modules/ROOT/examples/sensor-data-scala/src/main/resources/log4j.xml:18]:
```suggestion
</log4j:configuration>

```

**Comment by RayRoestenburg** [comment]
This is a quick fix. Making sure that log4j is not added to the classpath is an open issue: #620

**Pull request merged**